src/components/footer.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { FaInstagram, FaTwitterSquare, FaFacebookSquare } from "react-icons/fa";
+
+const SkipLink = styled.a`
+  position: absolute;
+  top: -1000px;
+  left: -1000px;
+  height: 1px;
+  width: 1px;
+  text-align: left;
+  overflow: hidden;
+
+  &:active,
+  &:focus,
+  &:hover {
+    left: 0;
+    top: 0;
+    width: auto;
+    height: auto;
+    overflow: visible;
+  }
+`;
+
+const Footer = () => {
+  const [state, setState] = useState(false);
```

Comment:
Are you going to be using these?

Reply:
Ok, removed the unused imports, skiplink, and useState. No errors this time! 😜